templates/aws-stack.yml

```diff
@@ -933,6 +945,8 @@ Resources:
           LAMBDA_TIMEOUT:        "50s"
           LAMBDA_INTERVAL:       "10s"
 
+  # TODO: give lambda permission to decrypt chamber SSM key
```

Comment:
So if the SSM var is encrypted, I think giving permission to the EC2 host would be done by users of this template using the `ManagedPolicyARN` parameter.  
However, nothing similar exists for the lambda. I assume that needs to be managed a similar way?